docker-compose.production.yml

```diff
@@ -4,7 +4,7 @@
 # Some other recommendations:
 # 1. To persist Postgres data, assign it a volume host location.
 # 2. Split the worker service to adhoc workers and scheduled queries workers.
-version: '2'
+version: '3'
```

Comment:
If there is no change in functionality I wonder if maybe it's worth to keep it as "version: '2'" to have wider range of supported Docker Compose versions?

Reply:
in your file there is no api change at all, v3 add more stuff to manage deploy and networking.
you can check it here:
https://docs.docker.com/compose/compose-file/compose-versioning/#upgrading

changing it adhoc after a clone back and forth is no brainer in this case, but housekeeping in my opinion is always good specially considering compose is in v3.7 already.

Aaaand considering this is a prod env, having the latest patch in the infra in your first deploy is a good practice, deploying a new prod server with outdated stuff must be avoided even knowning its a reality. 💅 

but feel free reject this pr since is a easy change back and forth.  😃